spec/support/solidus_stripe/backend_test_helper.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+module SolidusStripe::BackendTestHelper
+  def payment_method
+    # Memoize the payment method id to avoid fetching it multiple times
+    @payment_method ||= create(:stripe_payment_method)
+  end
+
+  def order
+    @order ||= create(:completed_order_with_totals, line_items_price: 50)
+  end
+
+  # Stripe-related helper methods for creating and fetching Stripe objects
+
+  def create_stripe_payment_method(card_number)
+    payment_method.gateway.request do
+      Stripe::PaymentMethod.create({
+        type: 'card',
+        card: {
+          number: card_number,
+          exp_month: 12,
+          exp_year: (Time.zone.now.year + 1),
+          cvc: '123',
+        },
+      })
+    end
+  end
+
+  def create_payment_intent(stripe_payment_method_id)
```

Comment:
```suggestion
  def create_stripe_payment_intent(stripe_payment_method_id)
```

For consistency :slightly_smiling_face: